agent/php_execute.c

```diff
@@ -1035,7 +1035,8 @@ static void nr_php_execute_metadata_add_code_level_metrics(
 
 #undef CHK_CLM_STRLEN
 
-  lineno = nr_php_zend_execute_data_lineno(execute_data);
+  // lineno = nr_php_zend_execute_data_lineno(execute_data);
```

Comment:
```suggestion
```

Reply:
Actually, that line needs to be totally removed.
[c24df84](https://github.com/newrelic/newrelic-php-agent/pull/558/commits/c24df8444ac9b0a9c43d49b85225ce09a7ea1322)